src/inflators/model-loader.ts

```diff
@@ -0,0 +1,16 @@
+import { HubsWorld } from "../app";
+import { inflateMediaLoader } from "./media-loader";
+
+export type ModelLoaderParams = {
+  src: string;
+}
+
+export function inflateModelLoader(world: HubsWorld, eid: number, params: ModelLoaderParams) {
+  inflateMediaLoader(world, eid, {
```

Comment:
Based on the old defaults https://github.com/mozilla/hubs/blob/aaa4584a822545a8ea998c3c14edc62078fb542b/src/gltf-component-mappings.js#L267-L275 I think resize and recenter should be true. Technically models like this had an object menu too, though it was a restricted one with only "open link", "refrresh" , "inspect', and "focus" - I don't think its particularly important to preserve that.